{"name": "test_landslide_case_start", "status": "failed", "statusDetails": {"message": "AssertionError: landslide is running a case!\nassert False", "trace": "get_test_id = False\n\n    def test_landslide_case_start(get_test_id):\n>       assert get_test_id, \"landslide is running a case!\"\nE       AssertionError: landslide is running a case!\nE       assert False\n\ntest_001_pdu_establishment.py:48: AssertionError"}, "attachments": [{"name": "log", "source": "e06cf303-4309-475d-80b1-dc4e7003096a-attachment.txt", "type": "text/plain"}], "start": 1566877021720, "stop": 1566877021721, "uuid": "c5ef6019-acd0-4f42-81e9-b6a83ad3f741", "historyId": "9eb2b9f80043f88b2f58d008f2727442", "testCaseId": "b3f4749ff5dc127898a5c843642053f0", "fullName": "test_001_pdu_establishment#test_landslide_case_start", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_001_pdu_establishment"}, {"name": "subSuite"}, {"name": "host", "value": "ubuntu18"}, {"name": "thread", "value": "29919-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_001_pdu_establishment"}]}